mkosi/distributions/debian.py

```diff
@@ -119,6 +115,13 @@ def install_packages(cls, state: MkosiState, packages: Sequence[str], apivfs: bo
 
         policyrcd.unlink()
 
+        for d in state.root.glob("boot/vmlinuz-*"):
+            kver = d.name.removeprefix("vmlinuz-")
+            vmlinuz = state.root / "usr/lib/modules" / kver / "vmlinuz"
+            if not vmlinuz.exists():
+                shutil.move(d, vmlinuz)
```

Comment:
Should this maybe be a copy instead of the move so as not to break stuff that depends on the original location? A symlink might be a solution too, if `/boot/` supports it.